### PR TITLE
Fix missing content with new Hugo.

### PR DESCRIPTION
**Pull request by dj2**
# Summary | Résumé

It seems with newer Hugo the main page needs to use `.Site.Pages`
instead of `.Pages` in order for the page content to show up.

This seems to also pull in the folder as a data item, it's fine for the
products as they have an `onhomepage` filter. For the blog posts I added
in a filter to make sure there was a `author` tag in order to filter
out the folder.

Issue #2052

### CODE REVIEW

File: layouts/index.html [hunk 1]
```diff
@@ -87,7 +87,8 @@ <h2>{{i18n "featured-post" }}</h2>
         </div>
       </div>
       <div class="row blog-posts">
-        {{ $posts := where .Pages "Section" "blog" }}
+        {{ $p := where .Site.Pages "Section" "==" "blog" }}
+        {{ $posts := where $p ".Params.processed" ">" 0 }}
```
Comment (brdunfield) [inline layouts/index.html:91]: The processed tag is a legacy tag from a previous pipeline. It's not present in strapi (and any blog posts being managed there, which are from the last few months and any future ones). So this probably won't work :(

Comment (dj2) [reply layouts/index.html:91]: Nice catch, thanks. I'll see if I can find another key which can be used for this purpose.

Does that mean I could make a PR to remove all the `processed` entries? Are there other keys which can be removed?

Comment (brdunfield) [reply layouts/index.html:91]: Yep, the processed keys are not being used. I would have to take a more in depth look to remember any others, but I can probably get back to you on that later today.

Comment (dj2) [reply layouts/index.html]: Ok, switched to `.Params.author != nil` as the folder entry doesn't have an author but as far as I can tell all blog posts do.

**Review by brdunfield — DISMISSED** [review]
LGTM - content on the homepage looks the same as the live site!

**Review by brdunfield — APPROVED** [review]
👍

**Pull request merged**